src/src/AdminBundle/Resources/views/Event/edit.html.twig

```diff
@@ -10,4 +10,87 @@
             {{ form_end(form) }}
         </div>
     </div>
+    <div class="row">
+        <h3>Template Testing</h3>
+        <div class="col-md-6">
+            <p>sem pride form s emailovou adresou</p>
```

Comment:
daj tma aspon todo a napis to po anglicky

Reply:
oki